Algorithm/QCAlgorithm.Python.cs

```diff
@@ -298,7 +301,7 @@ public void AddUniverse(Type dataType, SecurityType securityType, string name, R
             var marketHoursDbEntry = MarketHoursDatabase.GetEntry(market, name, securityType);
             var dataTimeZone = marketHoursDbEntry.DataTimeZone;
             var exchangeTimeZone = marketHoursDbEntry.ExchangeHours.TimeZone;
-            var symbol = QuantConnect.Symbol.Create(name, securityType, market);
+            var symbol = QuantConnect.Symbol.Create(name, securityType, market, null, dataType);
```

Comment:
nit - Could avoid sending in `null` specifying `baseDataType:`

Reply:
I can dig it 👍